app/views/responsiblepeople/detailed_answers.scala.html

```diff
@@ -405,10 +420,10 @@
                             </ul>
                         }
                     }
-                    <p>@ex match {
-                    case ExperienceTrainingYes(info) => {<p>@Messages("lbl.yes")</p> @info}
+                    @ex match {
+                        case ExperienceTrainingYes(info) => {<p>@Messages("lbl.yes")</p> <p>@info</p>}
```

Comment:
Too many <p> elements? (Or is this the one we resolved earlier?)

Reply:
This is the one we resolved earlier